impexp/updateData.js

```diff
@@ -1,14 +1,18 @@
-// This runs the code to update the json data from a csv file
-//
-// updateData.sh inputFileName
-//
-var UcumDataUpdater = require("./ucumDataUpdater.js").UcumDataUpdater;
-
-var inputFileName = process.argv[2];
-if (inputFileName) {
-  var upd = UcumDataUpdater.getInstance();
-  upd.readFile(inputFileName);
+/**
+ * This is the tool for generating ucumDefs.json. See the README for details.
+ */
+
+import { UcumXmlDocument } from "../source/ucumXmlDocument.js";
```

Comment:
I would feel more confident that the existing ucumDefs.json was not affecting this if this program first deleted ucumDefs.json before any imports or requires.  (I think import has to be at the beginning, so you might need to switch to require.)

Reply:
Even if I'm sure that the presence of this file does not affect the process now?